src/Kernel/Constants/Rights.cs

```diff
@@ -2,19 +2,19 @@
 {
   /// <summary>
   /// Rights give access to admin rights to the corresponding services or blocks
-  /// Admin, Feddback, Email, create/update Roles only work with admin rights
+  /// Admin, Feedback, Email, create/update Roles only work with admin rights
   /// </summary>
-  /// <param name="AddEditRemoveUsers">Right grants admin rights to user, family, education, skill, office users, achievement </param>
-  /// <param name="AddEditRemoveProjects">Right grants admin rights to project, task services</param>
-  /// <param name="AddEditRemoveEmailsTemplates">Right grants admin rights to text template service</param>
-  /// <param name="AddEditRemoveDepartments">Right grants admin rights to department service, work with project department</param> 
-  /// <param name="AddEditRemoveNews">Right grants admin rights to news service</param> 
-  /// <param name="AddEditRemovePositions">Right grants admin rights to position service</param> 
-  /// <param name="AddEditRemoveTime">Right grants admin rights to time service</param> 
-  /// <param name="AddEditRemoveCompanies">Right grants admin rights to company, office services</param> 
-  /// <param name="AddEditRemoveCompanyData">Right grants admin rights to company service to work with company users, office workspaces/workspaceTypes, booking service</param> 
-  /// <param name="AddRemoveUsersRoles">Right grants admin rights to right service to work with users roles</param> 
-  /// <param name="AddEditRemoveWiki">Right grants admin rights to wiki service</param>
+  /// <field name="AddEditRemoveUsers">Right grants admin rights to user, family, education, skill, office users, achievement </field>
```

Comment:
may be stay 'param'?
https://learn.microsoft.com/ru-ru/dotnet/csharp/language-reference/xmldoc/recommended-tags
https://habr.com/ru/post/41514/

Reply:
maybe <value> then?

Reply:
'value' - OK;
and, please, add descriptions for AddEditRemoveOffices and AddEditRemoveBooking

Reply:
and fix Companies+CompanyData if needed